docker-compose.yml

```diff
@@ -1,11 +1,37 @@
-app:
-  mem_limit: 300m
-  memswap_limit: 1g
-  env_file:
-    .env
-  build: .
-  ports:
-    - '3000:8000'
-  volumes:
-    - .:/home/nodejs/app
-    - /home/nodejs/app/node_modules
+version: '2'
+
+services:
+  mongodb:
+    container_name: ya-mongo-db
+    image: alexpunct/mongo:3.4
+    volumes:
+      - mongostorage:/data/db
+    ports:
+      - "${MONGODB_EXPOSED_PORT}:27017"
+    environment:
+      - MONGO_DATA_DIR=/data/db
+      - MONGO_LOG_DIR=/dev/null
+      - MONGODB_ADMIN_USER=${MONGODB_ADMIN_USER}
+      - MONGODB_ADMIN_PASS=${MONGODB_ADMIN_PASS}
+      - MONGODB_APPLICATION_DATABASE=${MONGODB_APPLICATION_DATABASE}
+      - MONGODB_APPLICATION_USER=${MONGODB_APPLICATION_USER}
+      - MONGODB_APPLICATION_PASS=${MONGODB_APPLICATION_PASS}
+
+  api:
+    container_name: ya-server
+    build: .
+    restart: always
+    mem_limit: 300m
```

Comment:
Is that enough?

Reply:
@markelog I was following docker guidelines/recommendations, maybe we can discuss this

Reply:
mmm, do you have a link?

Reply:
@markelog https://github.com/oleg-koval/docker-node
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#build-cache